packages/core/event.d.ts

```diff
@@ -1,4 +1,5 @@
-import { Event } from '.'
+import { App, Device, Event, Request, Breadcrumb, User, Session } from './types'
+import { Error } from './types/event'
```

Comment:
I've not added `Error` to the public facing type exports but we may want to take another look at how the types are organised and exported at some point for consistency.